amethyst_utils/src/screen_to_world.rs

```diff
@@ -0,0 +1,25 @@
+//! Provides utilities for transform position.
+
+use amethyst_core::nalgebra::{Matrix4, Orthographic3, Vector3};
+use amethyst_renderer::ScreenDimensions;
+
+/// Transforms position from screen space into world space
+pub fn screen_to_world(
+    screen_position: (f64, f64),
+    camera_translation: &Vector3<f32>,
+    camera_proj: &Matrix4<f32>,
```

Comment:
Wouldn't it make more sense here to directly take the Orthographic3 struct here?
It makes the user aware of what the parameter is, gives them the responsibility of providing a good projection and saves a cast when they already have the Orthographic3 struct.

Reply:
Perhaps. I have little experience in development.
In my case it allows me not to write `use amethyst::core::::nalgebra::Orthographic3`.
And I probably would not understand where to get it, if I saw it the first time.
Right now, it’s enough for the user to pass `camera.proj` without any conversions.

Reply:
Well, I think the real issue here is that camera.proj is a Matrix4 and not an Orthographic3, as Orthographic3 anyway is just a wrapper for Matrix4. Let's leave it as is for now, it should be improved in the new renderer.

Reply:
May be this should to take a link to the camera object for greater clarity

Reply:
Good idea. What you can do is in the doc just above explain that the matrix is the camera projection matrix field, or an Orthographic3 matrix.

Reply:
Do you have an example of how I should format this?